docs/reference/agent-config.txt

```diff
@@ -0,0 +1,45 @@
+.. _agent-configuration:
+
+Agent Configuration
+===================
+
+For both the master and the agent, configuration options may be set using a
+configuration file, environment variables, or command-line options. (Although
+values from different sources will be merged, we generally recommend sticking
+to a single source for each service to keep things simple.)
+
+Each option in the configuration file corresponds directly to an
+environment variable or command-line option. For example, the agent
+configuration file might contain
+
+.. code:: yaml
+
+  master_address: 0.0.0.0
+  master_port: 8080
+
+
+Required Options
+----------------
+The following options are required for connecting an agent to the Determined
+master.
+
+Master Address
+~~~~~~~~~~~~~~
+The address of the Determined master can be configured by setting the ``master_address``
+option.
+
+Master Port
+~~~~~~~~~~~
+The port of the Determined master located at ``master_address``. This can be configured
+via the ``master_port`` option.
+
+Agent Configuration
```

Comment:
This results in two top-level headings, both named "Agent Configuration". Rename this perhaps?